homeassistant/components/shelly/sensor.py

```diff
@@ -961,6 +961,12 @@ class RestSensorDescription(RestEntityDescription, SensorEntityDescription):
         device_class=SensorDeviceClass.BATTERY,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    "analoginput_xpercent": RpcSensorDescription(
+        key="input",
+        sub_key="xpercent",
+        name="Analog value",
```

Comment:
I'm not sure about the name of the sensor, maybe someone has a better idea?

Reply:
How is called in Shelly app (if is present)

Reply:
I don't know, I don't use Shelly app.